string event_id
target_gps[] data